### DISCUSSION TITLE
UK support

### DISCUSSION THREAD

**Discussion by OptimusGREEN** [dahlb/ha_kia_hyundai#76] [General]
Hi,

I just got my new kia and would love to implement it in Home Assistant.

Will UK support be coming soon?

cheers

**Comment by dahlb** [comment]
UK I think falls under EU regional which this integration doesn't support due to region implementation limitations, but this fork has support, try https://github.com/Hyundai-Kia-Connect/kia_uvo

**Comment by OptimusGREEN** [comment]
Great thanks
